data/raw/state_renewable_gen_targets.csv

```diff
@@ -1,50 +1,50 @@
-FL,207720.32,136592.29,12576.86,192893.46,146151.54,17844.47,356889.47
-GA,73295.48,90859.78,29054,98420.52,85128.29,9660.46,193209.26
-ID,4177.18,11977.31,11763.43,12178.73,14343.29,1395.9,27917.91
-IL,56068.98,70812.65,82031.14,92753.62,105713.52,10445.64,208912.78
-IN,82210.88,52878.55,39816.29,85447.75,80712.68,8745.29,174905.71
-IA,21294.77,21788.51,23182.71,30186.96,32765.73,3313.3,66265.99
-KS,19836.86,21130.18,14527,26599.43,26119.91,2774.7,55494.04
-KY,58082.69,35507.2,20189.71,57127.56,50963.06,5688.98,113779.6
-LA,79860.07,27649.75,7758.86,61302.88,48202.36,5763.43,115268.67
-ME,1996.6,10129.75,11607,9919.45,12627.23,1186.67,23733.34
-MD,17633.97,38575.85,27062.14,38492.81,40615.57,4163.6,83271.97
-MA,14213.52,37683.93,44060.71,40880.59,50279.66,4797.91,95958.16
-MI,66297.19,65429.24,70851.71,92288.01,100161.22,10128.91,202578.14
-MN,25882.79,40518.98,48086.14,49985.1,58778.43,5724.4,114487.92
-MS,58263.52,26088.87,5913,48049.46,37702.67,4513.27,90265.4
-MO,59404.22,49693.09,31911.43,68938.72,65019.58,7050.44,141008.74
-MT,9249.97,7347.78,9020.71,11654.56,12682.98,1280.92,25618.47
-NE,20011.39,14122.18,13870,22657.06,22946.33,2400.18,48003.57
-NV,28760.79,18772.95,9218.86,28724.83,25190.13,2837.63,56752.59
-NH,3808.18,8760,10324.29,9803.3,11944.54,1144.62,22892.47
-NJ,31811.39,46932.36,47867.14,56711.86,63568.49,6330.54,126610.9
-NM,24822.61,17466.91,9802.86,26004.03,23483.72,2604.62,52092.37
-NY,53458.94,88810.47,119720,111769.77,137120.16,13099.47,261989.41
-NC,62311.2,78680.73,35665.71,87531.96,80292.8,8832.88,176657.64
-ND,26053.64,5383.42,7049.71,19264.3,17298.13,1924.34,38486.77
-OH,99268.41,73138.04,70121.71,114457.63,115944.13,12126.41,242528.16
-OK,49722.85,34317.96,14746,50351.33,43496.15,4939.34,98786.82
-OR,20652.4,29146.91,20992.71,33267.58,33984.84,3539.6,70792.02
-PA,144866.3,70961.31,76493.57,140123.39,137581.74,14616.06,292321.18
-RI,7395.13,5383.42,7049.71,9002.12,9834.73,991.41,19828.27
-SC,36751.17,39743.85,12326.57,45523.7,38856.81,4441.08,88821.6
-SD,2758.08,5861.24,7675.43,6889.74,8590.26,814.74,16294.75
-TN,30032.6,50797.38,26864,51978.41,50330.87,5384.7,107693.98
-TX,328207.39,206608.58,56126.57,309864.23,251531.19,29547.13,590942.55
-UT,32313.11,20737.31,20367,34880.49,34866.06,3670.87,73417.42
-VT,42.74,5160.44,5068.29,4280.87,5477.02,513.57,10271.46
-VA,66616.81,59249.45,39420,80264.05,76757.91,8264.31,165286.26
-WA,19495.65,45297.16,42840.57,47631.4,54620.31,5381.67,107633.38
-WV,53182.04,12561.31,8812.14,38626.24,32201.48,3727.77,74555.49
-WI,45868.54,40773.82,46720,60734.9,65959.34,6668.12,133362.36
-WY,35168.22,6434.62,4213.14,24061.24,19463.94,2290.8,45815.98
\ No newline at end of file
+state,rnw_generation_fossil_fuels,rnw_generation_transportation,rnw_generation_buildings,total_gen_by_solar,total_gen_by_wind,total_gen_by_other,total_rnw_gen_needed,current_solar,current_wind,perc_solar_target,perc_wind_target
+AL,76967.78,40986.18,14089,68819.6,56621.21,"6,602","132,043",383.58,0,1,0

```

Comment:
@swatland610 Alaska is missing here, which is currently causing the build to fail due to missing data